std/src/main/scala/algebra/std/boolean.scala

```diff
@@ -9,6 +10,13 @@ package object boolean extends BooleanInstances
 trait BooleanInstances {
   implicit val booleanAlgebra: BooleanAlgebra =
     new BooleanAlgebra
+
+  val BooleanRing = new BoolRing[Boolean] {
```

Comment:
Hmm, so `BoolRing` requires `a+a = 0`, thus specifically `1+1 = 0`, so only fields of characteristic 2 are boolean rings.

On the other side, `Field` requires a multiplicative inverse for every element except 0, i.e. `∀a≠0 ∃b a⋅b = 1`, which rewritten to lattice notation is `∀a≠0 ∃b a∧b = 1`, but this is only true for `a=1`. Thus any boolean ring with more than two elements will not be a field.

If the same operations are already implemented, I can remove this instance and make the `Field` instance extend `BoolRing`.